components/slider/index.css

```diff
@@ -10,63 +10,95 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-slider.css";
 
+
+/* missing tokens for variant with Tick, also need t-shirt sizing */
+.spectrum--medium {
+  --spectrum-slider-tick-mark-height: 10px;
+  /* --spectrum-global-dimension-size-125 */
+  --spectrum-slider-tick-mark-width: 2px;
+  /* --spectrum-global-dimension-size-25: */
+  --spectrum-slider-tick-mark-border-radius: 2px;
+  /* --spectrum-global-dimension-size-25 2px for Spectrum,   --spectrum-global-dimension-size-10 1px for Express */
+  --spectrum-slider-ramp-track-height: 16px;
+  /* dimension-size-200 */
+}
+
+.spectrum--large {
+  --spectrum-slider-tick-mark-height: 13px;
+  --spectrum-slider-tick-mark-width: 2px;
+  --spectrum-slider-tick-mark-border-radius: 2px;
+  --spectrum-slider-ramp-track-height: 20px;
+  /* dimension-size-200 */
+}
+
+.spectrum-Slider--sizeS {
+
+  --spectrum-slider-handle-size: var(--mod-slider-handle-size-small, var(--spectrum-slider-handle-size-small));
```

Comment:
@pfulton let me know if we still need those -mods for items that already have the -mod options from previous assignments.

Reply:
Wow, I completely missed this question, and I apologize for not seeing it and addressing it until now.

Overall, I'd like us to avoid declaring `--mod` custom property overrides in initial declarations and to override "global" tokens.

Avoid this:
```css
.spectrum-Slider {
  --spectrum-slider-handle-border-radius: var(--mod-corner-radius-200, var(--spectrum-corner-radius-200));}
```

Instead, do something like this:
```css
.spectrum-Slider {
  --spectrum-slider-handle-border-radius: var(--spectrum-corner-radius-200);}

/* later, when it's actually used: */
  border-radius: var( --mod-slider-handle-border-radius, var(--spectrum-slider-handle-border-radius));
```

The implications are a bit nuanced here, but due to the cascade, it should be a little safer to do something like I did in my suggestion above. A `--mod-corner-radius-200` could potentially be more wide-reaching than `--mod-slider-handle-border-radius`.

Plus, IMO, this gets things a bit more inline with the "API" we're providing to consumers for a given component (they're "modifying" Slider related custom properties rather than a "global" Spectrum corner radius).

That's a big explanation, but I think we should change the approach here.